src/app/users/view-user/view-user.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container m-b-20" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="end" fxLayoutGap="2%">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+  <button mat-raised-button color="primary">
+    <fa-icon icon="cog"></fa-icon>&nbsp;&nbsp;
+    Change Password
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
+          Login Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ user.username }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          First Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ user.firstname }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Last Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ user.lastname }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Email
+        </div>
+
+        <div fxFlex="50%">
+          {{ user.email }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Office
+        </div>
+
+        <div fxFlex="50%">
+          {{ user.officeName }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Roles
+        </div>
+
+        <div fxFlex="50%">
+          <div *ngFor="let role of user.selectedRoles">
```

Comment:
```suggestion
          <span *ngFor="let role of user.selectedRoles">
```
Lets just use a span here.